src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -86,22 +93,33 @@ public function supportsNormalization($data, $format = null)
     /**
      * {@inheritdoc}
      *
+     * In PHP, The $timezone parameter and the current timezone are ignored when the $time parameter either is a UNIX timestamp (e.g. @946684800) or specifies a timezone (e.g. 2010-01-28T15:00:00+02:00).
+     *
+     * This denormalizer assumes that all DateTimeInterface object returned will have the timezone returned by the getTimezone() method.
+     * Default PHP behavior will occur if the getTimezone() method returns null or $defaultContext[self::PRESERVE_CONTEXT_TIMEZONE] is set to false.
```

Comment:
I would move this explanation to the new flag directly.

Reply:
Done